packages/npm-dependency-fetcher/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@dependency-health-monitor/npm-dependency-fetcher",
+  "version": "1.0.0",
+  "description": "NPM dependency fetcher that is able to fetch in depth",
+  "main": "dist/index.js",
+  "type": "module",
+  "scripts": {
+    "build": "tsc",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Change to MIT licence

Reply:
Done d8a01010c24c9eb7c28e94ccac8176b5f7b80b0f